content/en/docs/reference/glossary/Quantity.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Quantity
+id: quantity
+date: 2023-01-24
+short_description: >
+ Quantity contains GB and GiB. 
+
+aka: 
+tags:
+- fundamental
+---
+
+"Quantity" refers to a specific amount of a resource, such as CPU or memory, that is allocated to a container or pod. 
+For example, a pod may be allocated 1 CPU core and 2 gigabytes of memory, which would be specified as "1 CPU" and "2Gi" for the respective resources. These quantities are used by the Kubernetes scheduler to determine which nodes have the available resources to run a pod, and to ensure that pods do not consume more resources than they are allocated.
```

Comment:
```suggestion
For example, a pod may be allocated 1 CPU core and 2 Gigabytes of memory, which would be specified as "1 CPU" and "2Gi" for the respective resources. These quantities are used by the Kubernetes scheduler to determine which nodes have the available resources to run a pod, and to ensure that pods do not consume more resources than they are allocated.
```

I feel like this text could be improved a lot. If we're mentioning about the use of scheduler, it'd be good if we can provide a reference link for the users to go through [docs](https://kubernetes.io/docs/concepts/scheduling-eviction/kube-scheduler/) for in-depth information. WDYT?

Reply:
great idea, will add that?

Reply:
This is not implemented.